src/main/webapp/WEB-INF/views/car/addDriverToCar.jsp

```diff
@@ -0,0 +1,21 @@
+<%--
+  Created by IntelliJ IDEA.
+  User: Mark
+  Date: 24.11.2022
+  Time: 9:28
+  To change this template use File | Settings | File Templates.
+--%>
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Add Driver to Car</title>
+</head>
+<body>
+  <h1>Enter cars id and Drivers id</h1>
+  <form method="post" name="${pageContext.request.contextPath}/car">
+    Car id <input type="text" name="car_id">
+    Driver id <input type="text" name="driver_id">
```

Comment:
All checks to see if any field is empty or that field should accept only numbers should be done by JSP. HINT: use JSP input tag [properties](https://www.w3schools.com/html/html_form_input_types.asp)

Reply:
@ana-rom also, you shouldn't approve solution if it has common mistakes

Reply:
@hronograf sorry, got it